app/src/main/java/com/infomaniak/mail/ui/main/folder/TwoPaneFragment.kt

```diff
@@ -75,6 +75,11 @@ open class TwoPaneFragment : Fragment() {
                 R.string.noConversationSelected,
                 displayedFolder.getLocalizedName(requireContext()),
             )
+
+            if (displayedFolder.id != previousFolderId) {
+                previousFolderId = displayedFolder.id
+                if (isInThread) closeThread()
+            }
```

Comment:
We should probably poll to know if we want to do this or not

Reply:
Done. We keep closing it.